### PR TITLE
Auto send set on customer

### PR DESCRIPTION
**Pull request by kaleposhobios**
What do you think? Should there be an explicit boolean to auto-send or not, inside that payload?

### CODE REVIEW

File: brokers/rest.md [hunk 1]
```diff
@@ -816,6 +816,11 @@ Request:
           "generate": true,                                    // Optional. If omitted, leaves unchanged.
           "template_name": "generic_template.doc"              // Optional. Needs to exist if generate is true.
       },
+      "auto_send_invoices": {                                  // Optional. If omitted will not auto_send. If passed, will set to auto-send. 
+        "interval": "weekly",
+        "at": 4,
+        "recipient": "user@example.com"
```
Comment (r38y) [inline brokers/rest.md:822]: Maybe make this an array instead? I've been on a kick to get data to be more structured and then we can manipulate it however we want on the backend.

**Review by r38y — APPROVED** [review]

File: brokers/rest.md [hunk 2]
```diff
@@ -816,6 +818,11 @@ Request:
           "generate": true,                                    // Optional. If omitted, leaves unchanged.
           "template_name": "generic_template.doc"              // Optional. Needs to exist if generate is true.
       },
+      "auto_send_invoices": {                                  // Optional. If omitted will not auto_send. If passed, will set to auto-send. 
+        "interval": "weekly",                                  // See `auto_send_invoices` values below
```
Comment (nathanmadams) [inline brokers/rest.md:822]: Is this needed given that there's already a `billing_interval` field in the payload?

File: brokers/rest.md [hunk 3]
```diff
@@ -816,6 +818,11 @@ Request:
...
+        "at": 4,                                               // Day of week or day of month to send on (a little before midnight on this daty)
+        "print_user_email": "user@example.com"                 // For any invoices set to deliver as "print", what user to email them to.
```
Comment (nathanmadams) [inline brokers/rest.md:824]: This has to be the email address for a current HubTran user correct?

File: brokers/rest.md (hunk 2 above)
Comment (kaleposhobios) [reply brokers/rest.md:822]: No this is irrelevant. I wasn't thinking.

File: brokers/rest.md (hunk 3 above)
Comment (kaleposhobios) [reply brokers/rest.md:824]: Currently yes. At the time I was thinking of relaxing that requirement. Now that you're digging into the requirements and implementation let me know if you agree.

Comment (nathanmadams) [reply brokers/rest.md:824]: Will Acertus be using the print delivery_method at all?

File: brokers/rest.md (hunk 2 above)
Comment (nathanmadams) [reply brokers/rest.md:822]: OK I'll update this diff accordingly

File: brokers/rest.md [hunk 4]
```diff
@@ -852,9 +856,13 @@ Response:
         "city": "city",
         "state": "state",
         "postal_code": "12345",
-        "country": "US"
+        "country": "US",
+        "phone": "4039991111",
```
[the remark this answers is not in the document]
Comment (nathanmadams) [reply brokers/rest.md:860]: Will do. Did we not end up adding those fields?

[the remark this answers is not in the document]
Comment (kaleposhobios) [reply brokers/rest.md]: https://github.com/HubTran/documentation/pull/95

**Comment by kaleposhobios** [comment]
Cool! I can't approve this because I'm the creator. Is this finalized to share with customers?

**Comment by nathanmadams** [comment]
@kaleposhobios Yes I think we can share this. I removed the "print_user_email" field but we can add that back in later if Acertus needs the ability to control that at the customer level. I was thinking managing that at the account level might be good enough. But if you've already had those discussions with Acertus and determined they need it I can put it back.

**Comment by kaleposhobios** [comment]
Thanks!

**Pull request merged**